lti_consumer/lti_xblock.py

```diff
@@ -896,6 +896,10 @@ def author_view(self, context):
         # Retrieve LTI 1.3 Launch information
         context.update(get_lti_1p3_launch_info(block=self))
 
+        context.update({
+            'launch_preview_url': self.runtime.handler_url(self, 'lti_1p3_launch_handler').rstrip('/?')
```

Comment:
Didn't expect a URL ending in `?`. Is it necessary?

Reply:
@viadanna I've copied it from [here](https://github.com/edx/xblock-lti-consumer/blob/9ac5fda45c352fc456ab8b432bb967faf1a4661a/lti_consumer/lti_xblock.py#L1363). Not sure why that's there. I thought it might be there for some kind of issue before. Should I remove it?

Reply:
In that case, let's keep it, I might be missing some context here.